demo/src/config/index.js

```diff
@@ -28,7 +28,7 @@ const config = {
   urls: {
     octocatIcon: '/icons/octocat.png',
     themeModeIcon: '/icons/theme-mode.png',
-    IEditorOptions: 'https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.IEditorOptions.html#acceptSuggestionOnCommitCharacter',
+    IEditorOptions: 'https://microsoft.github.io/monaco-editor/typedoc/interfaces/editor.IDiffEditorConstructionOptions.html',
```

Comment:
I think this should be `https://microsoft.github.io/monaco-editor/typedoc/interfaces/editor.IEditorConstructionOptions.html`. So, `IEditorConstructionOptions` no `IDiffEditorConstructionOptions`. Isn't it?

Reply:
yeah you're right